src/services/time-slot.service.ts

```diff
@@ -39,6 +40,8 @@ export class TimeSlotService {
     return this.timeSlotRepository.insert({
       group: groupObjectId,
       ...parameters,
+      qrcodeSecret: crypto.randomUUID(),
```

Comment:
Can you use the same method to name an uploaded file here line 35 : https://github.com/SoWeQuiches/Backend/blob/main/src/services/aws.service.ts